src/ASM/LR/ASMu3D.h

```diff
@@ -16,6 +16,7 @@
 
 #include "ASMunstruct.h"
 #include "ASM3D.h"
+#include "LRSpline/LRSpline.h"
```

Comment:
i can do it, but at the cost of writing a custom deleter function for the shared ptr. i think that cost more than it was worth. but if you prefer it i can do it.